arrow-integrations-jackson-module/src/main/kotlin/arrow/integrations/jackson/module/NonEmptyListModule.kt

```diff
@@ -26,36 +31,37 @@ object NonEmptyListModule : SimpleModule(PackageVersion.VERSION) {
   }
 }
 
-private object NelSerializationConverter : StdConverter<Nel<*>, List<*>>() {
-  override fun convert(value: Nel<*>?): List<*>? = value?.all.orEmpty()
+class NonEmptyListSerializationConverter : StdConverter<NonEmptyList<*>, List<*>>() {
```

Comment:
Does `NonEmptyListSerializationConverter` require any kind of state that requires it to be a `class`? if not this could be an object but probably does not matter since this get's registered and instantiated only once?

Reply:
updated

Reply:
thanks!